### PR TITLE
update OWNERS, README, and Makefile `make images`

### PR DESCRIPTION
**Pull request by sallyom**
/assign @mfojtik 
/cc @deads2k 

update OWNERS and README and while we're at it let's fix `make images` target (currently broken, plus add the ability to pass a custom image tag)

### CODE REVIEW

**Comment by deads2k** [comment]
remove all previous owners

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,8 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # $2 - Dockerfile path
 # $3 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+IMAGE_TAG?=registry.svc.ci.openshift.org/ocp/4.3:cluster-policy-controller
```
Comment (soltysh) [inline Makefile]: Move those variables higher, that's the preferred approach, see https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/Makefile as an example.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
-##### cluster-policy-controller
+## cluster-policy-controller
+The cluster-policy-controller is responsible for maintaining policy resources necessary to create pods in a cluster. 
+Controllers managed by cluster-policy-controller are:
+* cluster quota reconcilion - manages cluster quota usage
+* namespace SCC allocation controller - allocates UIDs and SELinux labels for namespaces     
 
-TODO: Add content
+The `cluster-policy-controller` runs as a container in the `openshift-kube-controller-manager namespace`, in the kube-controller-manager static pod.
+This pod is defined and managed by the `kube-controller-manager`
```
Comment (soltysh) [inline README.md]: kube-controller-manager-operator, also add link to https://github.com/openshift/cluster-kube-controller-manager-operator/

File: Makefile (hunk 1 above)
Comment (sallyom) [reply Makefile]: I see, updated

File: README.md (hunk 2 above)
Comment (sallyom) [reply README.md]: added, thanks

File: README.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
-##### cluster-policy-controller
+## cluster-policy-controller
+The cluster-policy-controller is responsible for maintaining policy resources necessary to create pods in a cluster. 
+Controllers managed by cluster-policy-controller are:
+* cluster quota reconcilion - manages cluster quota usage
+* namespace SCC allocation controller - allocates UIDs and SELinux labels for namespaces     
 
-TODO: Add content
+The `cluster-policy-controller` runs as a container in the `openshift-kube-controller-manager namespace`, in the kube-controller-manager static pod.
+This pod is defined and managed by the [`kube-controller-manager`](https://github.com/openshift/cluster-kube-controller-manager-operator/)
```
Comment (soltysh) [inline README.md]: managed by the kube-controller-manager-operator

**Pull request merged**